eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -42,6 +42,7 @@ jobs:
     - ${{ if eq(parameters.platform, 'Linux_arm64') }}:
       - ${{ if or(eq(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true)) }}:
         - (Ubuntu.2110.Arm64.Open)Ubuntu.1804.ArmArch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-21.10-helix-arm64v8-20211116135000-0f8d97e
```

Comment:
Do we want to continue testing on Ubuntu 21.10 given that 22.04 is added to the matrix and that 21.10 will be EOL in three months (July 2022) before we even release .NET 7?

Reply:
@danmoseley do you remember if we intentionally left 20.04 out when we rehauled our test matrix last year?

Reply:
I don't see any indication that 20.04 is high risk so I would be fine not including it and dropping 21.10. @omajid does that sound fine?

Reply:
Yup, makes sense.

22.04 is higher-risk because it's one of the earliest Linux distros that adapted OpenSSL 3.0.